rainbowstream/draw.py

```diff
@@ -1066,21 +1066,27 @@ def format_quote(tweet):
     Quoting format
     """
     # Retrieve info
-    screen_name = '@' + tweet['user']['screen_name']
-    text = tweet['text']
+    screen_name = str( tweet['user']['screen_name'] )
+    text        = str( tweet['text'] )
+    tid         = str( tweet['id'] )
+
```

Comment:
Oops, sorry about that. The `str()` functions for `screen_name` and `text` are not really needed, since these values are already printable as is. For `tid` however, I should keep the `str()` since `tweet['id']` is an integer.